drv/transceivers-server/src/udp.rs

```diff
@@ -217,397 +215,572 @@ impl ServerImpl {
     fn handle_message(
         &mut self,
         msg: Message,
+        msg_id: u64,
         data: &[u8],
         tx_data_buf: &mut [u8],
     ) -> Option<u32> {
         // If the version is below the minimum committed, then we can't really
         // trust the message, even though it nominally deserialized correctly.
         // Don't reply.
-        if msg.header.version < version::MIN {
-            ringbuf_entry!(Trace::WrongVersion(msg.header.version));
+        if msg.version() < version::inner::MIN {
+            ringbuf_entry!(Trace::WrongVersion(msg.version()));
             return None;
         }
 
-        let (reply, data_len) = match msg.body {
+        // If the message is something we don't expect or something we cannot
+        // reply to
```

Comment:
Missing the back half of this sentence, maybe?

Reply:
I...don't know what I was going to say here. Removing.